packages/devtools_app/lib/src/http/http_request_data.dart

```diff
@@ -314,24 +314,25 @@ class TimelineHttpRequestData extends HttpRequestData {
   // as invalid.
   @override
   bool get isValid =>
-      _startEvent != null && !_startEvent.name.contains('HTTP CLIENT response');
+      _startEvent != null &&
+      !_startEvent!.name!.contains('HTTP CLIENT response');
 
   /// A map of general information associated with an HTTP request.
   @override
-  Map<String, dynamic> get general {
+  Map<String, dynamic>? get general {
```

Comment:
looks like this may be able to be non-null?

Reply:
nope. We return null if !isValid